src/app/backend/client/manager.go

```diff
@@ -20,7 +20,6 @@ import (
 	"log"
 	"strings"
 
-	"github.com/emicklei/go-restful"
```

Comment:
hmm...seems to be a bug if import name has a hyphen (https://github.com/joefitzgerald/go-plus/issues/52)